hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/NSSummaryEndpoint.java

```diff
@@ -116,21 +294,49 @@ public Response getDiskUsage(@QueryParam("path") String path,
     if (path == null || path.length() == 0) {
       return Response.status(Response.Status.BAD_REQUEST).build();
     }
+    return Response.ok(getDuResponse(path, listFile, withReplica)).build();
```

Comment:
After extracting the method here, but even before that, we can remove the @SuppressWarnings for the /du endpoint method in line 287